sdk/java-sdk/src/main/java/com/akkaserverless/javasdk/valueentity/ValueEntityProvider.java

```diff
@@ -35,4 +36,8 @@
   ValueEntityRouter<S, E> newRouter(ValueEntityContext context);
 
   Descriptors.FileDescriptor[] additionalDescriptors();
+
+  default Serializer serializer() {
+    return Serializer.noopSerializer();
+  }
```

Comment:
The Provider defines the Serializer to use. This is then passed to `AnySupport` and used before anything. 

`noopSeriazlier` has no effect and therefore `AnySupport` will just use protobuf.

Reply:
As a side idea, `AnySupport` could also be defined as a `Serializer`, but that's not in scope.